### DISCUSSION TITLE
HA database notifications not working

### DISCUSSION THREAD

**Discussion by dbnicholson** [requarks/wiki#3509] [Error / Bug Report]
**Describe the bug**
In HA mode with multiple containers events are not propagating through the database. I've used the current wiki docker image as well as dev HEAD (04a18968115b47f8eb7bbb64b3c7edabe93e817b) with an AWS RDS postgres database as well as a local postgres database. In both cases the postgres database is version 12. The postgres notification feature does work as I've connected with `psql` as well as a minimal pg-pubsub app and sent and received notifications.

It seems that the query is never sent to the database. I added several debug log statements and nothing is ever received by the listener even before the filtering by instance ID. The publish call is made in `notifyViaDB`, but neither the listener nor a manual `LISTEN wiki;` query ever receive anything. This obviously results in stale content being served when the request happens to go to the container that didn't do the local cache flush.

**To Reproduce**
1. Go to 'Admin -> Utilities -> Flush Cache'
2. Click on 'Flush Pages and Assets Cache'
3. See 'Cache flushed successly' message
4. Do not see `Received event flushCache from instance <random>: [ OK ]`
5. `/wiki/data/cache` directory in other container not empty

**Expected behavior**
Receive the `flushCache` event on all instances and clear the locally cached files.

**Host Info (please complete the following information):**
 - OS: Amazon Linux 2, Docker
 - Wiki.js version: 2.5.170
 - Database engine: postgres 12.3

**Comment by NGPixel** [comment]
Did you enable ha in all your nodes config.yml?

**Comment by dbnicholson** [comment]
Yes and I had debug logging on so I could see the HA activated message.

**Comment by jrose-stac** [comment]
Seeing the same issue with our setup. Running Wiki 2.5.170 on Amazon Linux 2, postgres 11.9. Our database instance is Aurora.

We see the same behavior with HA on and off in our setup.

**Comment by haydenseitz** [comment]
Have the same issue, posted details here: https://github.com/Requarks/wiki/discussions/3274#discussioncomment-2480498
